sdk/dotnet/Pulumi/Deployment/Deployment_RegisterResource.cs

```diff
@@ -95,6 +95,7 @@ private static void PopulateRequest(RegisterResourceRequest request, PrepareResu
                 request.AdditionalSecretOutputs.AddRange(customOpts.AdditionalSecretOutputs);
 
             request.IgnoreChanges.AddRange(options.IgnoreChanges);
+            request.ReplaceOnChanges.AddRange(options.ReplaceOnChanges);
```

Comment:
I would really like conformation here. I have been unable to test through a gRPC version change and don't speak C#.

Reply:
Thanks to @t0yv0 and @mikhailshilkov helping me, this has been tested and seems to work!